.mailmap

```diff
@@ -0,0 +1,6 @@
+Paweł Redzyński <pawelredzynski@gmail.com>
```

Comment:
Do we need this file? :slightly_smiling_face:

Reply:
not strictly, but it tidies correctly groups metadata for people who've used multiple emails/usernames. (e.g. running [`git fame`](https://github.com/casperdcl/git-fame) produces better results)

Reply:
Oh, this is actually first time I'm seeing anything like this. Is it used by git?

Reply:
sure https://www.git-scm.com/search/results?search=mailmap